0.preprocess-sites/4.image-and-segmentation-qc.py

```diff
@@ -164,31 +166,30 @@
     .reset_index()
     .rename(columns={0: "Ratio"})
 )
-
+empty_df=empty_df[['Percent_Empty']].stack().to_frame().reset_index().rename(columns={0: "Percent Empty"})
 quality_recode = {
     "Pass_Fail_withempty": "Pass/Fail (with empty)",
     "Pass_Fail_0empty": "Pass/Fail (without empty)",
-    "Percent_Empty": "Percent Empty",
 }
 ratio_df = ratio_df.assign(
     cell_quality_recode=ratio_df.Cell_Quality.replace(quality_recode)
 )
 
 ratio_gg = (
     gg.ggplot(ratio_df, gg.aes(x="x_loc", y="y_loc"))
-    + gg.geom_point(gg.aes(fill="Ratio"), size=5)
-    + gg.geom_text(gg.aes(label="site_location"), size=4, color="lightgrey")
-    + gg.facet_grid("cell_quality_recode~well", scales="free_y")
-    + gg.coord_fixed()
+    + gg.geom_point(gg.aes(fill="Ratio"), shape='s', size=6)
+    + gg.geom_text(gg.aes(label="site_location"), size=6, color="lightgrey")
+    + gg.facet_wrap("~cell_quality_recode + well", ncol = 3, scales="fixed")
```

Comment:
please make sure to run `black` on this script.

(Black will make sure there are no spaces in `ncol=3` 😄  )

Reply:
hahaha. I can't even count the number of times I told myself "run `black` before making the PR" and yet I still forgot!